### PR TITLE
+ testing invalid requests

### PR DESCRIPTION
**Pull request by aaabramov**
- ukrainian messages
- default capacity for InvalidRequestDto
- disabled shutdown hook
- some refactoring

### CODE REVIEW

File: web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java [hunk 1]
```diff
@@ -18,9 +17,10 @@
 public class InvalidRequestDto {
 
     private final List<FieldError> errors;
+    private static final int DEFAULT_CAPACITY = 3;
```
Comment (ghost) [inline web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java]: what's for?

File: web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+    @Test
+    public void testInvalidPatch() throws Exception {
+
+        final WarehouseDto warehouseDto = getRequestBody();
+        final String body = objectMapper.writeValueAsString(warehouseDto);
+
+        for (final Locale locale : availableLocales) {
+
+            final MockHttpServletRequestBuilder request = patch(Routes.WAREHOUSES + '/' + warehouseDto.getId())
```
Comment (ghost) [inline web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java]: implement route-wide test to cover all the needed controllers

Comment (aaabramov) [inline web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java]: shell I write for each controller their dto?
I was thinking about generalized test class where we could pass requestDto
and some other params and all common processes do in that class.

Exactly mine should test with invalid request bodies?

File: web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java (hunk 1 above)
Comment (aaabramov) [inline web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java]: it is ~average number of fields in out dtos.
Do we really need capacity = 10 ?

File: web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java (hunk 2 above)
Comment (ghost) [inline web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java]: think about something common - test to satisfy all the needed controllers

File: web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java (hunk 1 above)
Comment (ghost) [inline web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java]: i don't think so

File: web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java (hunk 2 above)
Comment (aaabramov) [inline web/src/test/java/edu/softserve/zoo/rest/request/RequestValidationTest.java]: hm.. these test is not for CRUD logic.
Also I think about writing test if there is custom i18ned validation message for every constraint in dtos.

Okay, I'll try to write those tests.

File: web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java (hunk 1 above)
Comment (aaabramov) [inline web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java]: don't think about 3 or 10? (

Comment (ghost) [inline web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java]: I don't think you need to use capacity in this case

Comment (aaabramov) [inline web/src/main/java/edu/softserve/zoo/dto/InvalidRequestDto.java]: @skyfenko clear, I'll fix that

**Pull request closed without merging**